EasyIntern-Backend/Models/JsonModelAccountLogin.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace EasyIntern_Backend.Models;
+
+public class JsonModelAccountLogin
+{
+    [Required]
+    public string Email { get; set; }
+    [Required]
+    public string Password { get; set; }
```

Comment:
If you want the password to be 8-13 chars long, add the [Range(8, 13)] attribute

Reply:
Interesting, but not relevant to the login dto